requirements.txt

```diff
@@ -1,5 +1,6 @@
+cleaning-scripts==0.2.16
```

Comment:
am I missing something or is this file used only for tests ? If that's the case, I think we can move all these dependencies to requirements-dev.txt